contrib/kubespray/roles/port/tasks/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: "Set host daemon port range"
+  sysctl:
+    name: net.ipv4.ip_local_port_range
+    value: "{{ host_daemon_port_start }} {{ host_daemon_port_end }}"
```

Comment:
if you edit `/etc/sysctl.conf` directly, do you need to restart network for the change to take effect by `/etc/rc.d/init.d/network restart`?

Reply:
ansible will run `/sbin/sysctl -p` automatically. Tested in bed, it take effect. Refer to https://docs.ansible.com/ansible/latest/modules/sysctl_module.html